easybuild/easyconfigs/i/IRkernel/IRkernel-1.2-foss-2021b-R-4.2.0.eb

```diff
@@ -0,0 +1,68 @@
+easyblock = 'Bundle'
+
+name = 'IRkernel'
+version = '1.2'
+versionsuffix = '-R-%(rver)s'
+
+homepage = 'https://irkernel.github.io'
+description = """The R kernel for the 'Jupyter' environment executes R code 
+ which the front-end (Jupyter Notebook or other front-ends) submits to the 
+ kernel via the network."""
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+dependencies = [
+    ('R', '4.2.0'),
+    ('Python', '3.9.6'),
+    ('IPython', '7.26.0'),
+    ('ZeroMQ', '4.3.4'),
+]
+
+exts_defaultclass = 'RPackage'
+exts_filter = ("R -q --no-save", "library(%(ext_name)s)")
+
+exts_default_options = {
+    'source_urls': [
+        'https://cran.r-project.org/src/contrib/',
+        'https://cran.rstudio.com/src/contrib/',
+        'https://cran.r-project.org/src/contrib/Archive/%(name)s/',
+    ],
+    'source_tmpl': '%(name)s_%(version)s.tar.gz',
+}
+
+exts_list = [
+    ('repr', '1.1.4', {
+        'checksums': ['6f799ca83e0940618dd8c22e62ffdce5ec11ba3366c5306ae58b55b53c097040'],
+    }),
+    ('IRdisplay', '1.1', {
+        'checksums': ['83eb030ff91f546cb647899f8aa3f5dc9fe163a89a981696447ea49cc98e8d2b'],
+    }),
+    ('pbdZMQ', '0.3-7', {
+        'checksums': ['df2d2be14b2f57a64d76cdda4c01fd1c3d9aa12221c63524c01c71849df11808'],
+    }),
+    (name, version, {
+        'checksums': ['5fb4dbdb741d05043120a8be0eb73f054b607d9854f314bd79cfec08d219ff91'],
+    }),
+]
+
+modextrapaths = {'R_LIBS_SITE': ''}
+
+# IPython notebook looks for the json kernel file in kernels/IRkernel
+# We start the kernel with default bitmapType 'cairo'. This is a more sensible default
+# for headless nodes. See https://github.com/IRkernel/IRkernel/issues/388
+local_kerneldir = '%(installdir)s/IRkernel'
+postinstallcmds = [
+    'mkdir -p %s/kernels/ir' % local_kerneldir,
+    'cp %s/kernelspec/* %s/kernels/ir' % (local_kerneldir, local_kerneldir),
+    ('sed -i \'s/"IRkernel::main()"/"options(bitmapType=\\x27cairo\\x27); IRkernel::main()"/g\''
+        ' %s/kernels/ir/kernel.json') % local_kerneldir
+]
+
+modextravars = {'JUPYTER_PATH': local_kerneldir}
```

Comment:
surely `JUPYTER_PATH` is a path-like variable, so should also be in `modextrapaths`?

Reply:
Yes, it should be in `modextrapaths`. Documentation is at https://docs.jupyter.org/en/latest/use/jupyter-directories.html#envvar-JUPYTER_PATH

I've also PR-ed 1.2 instead of 1.3.

Reply:
And #15776 for this same change in IRkernel 1.x